pkg/controller/reconciler/shared_test.go

```diff
@@ -43,8 +45,8 @@ const (
 	recipeTestControllerDelayInterval = time.Millisecond * 100
 )
 
-func createEnvironment(radius *mockRadiusClient, name string) {
-	id := fmt.Sprintf("/planes/radius/local/resourceGroups/%s/providers/Applications.Core/environments/%s", name, name)
+func createEnvironment(radius *mockRadiusClient, resourceGroup, name string) {
+	id := fmt.Sprintf("/planes/radius/local/resourceGroups/%s/providers/Applications.Core/environments/%s", resourceGroup, name)
```

Comment:
Just confirming, did we add the RG because we create one if it doesn't exist as part of the deployment template controller?

Reply:
yep, also just for clarity